src/apify/_actor.py

```diff
@@ -39,6 +39,8 @@
 MainReturnType = TypeVar('MainReturnType')
 
 
+@docs_name('Actor')
+@docs_group('Classes')
 class _ActorType:
```

Comment:
Thinking whether rather not to use just a one decorator with keyword arguments?

```suggestion
@docs(name='Actor', group='Classes')
class _ActorType:
```

Reply:
Sure, that's absolutely doable 👍🏽  It would require some more patching of the docusaurus plugin and I'd rather get these changes out now (because of the [broken links](https://github.com/apify/apify-docs/actions/runs/11924202390) this PR fixes). 

We can definitely put this on the todo list for the next iteration of the Python docs (together with putting this decorator in `apify-shared`... or somewhere, as I'm mentioning in the PR description). 

If you feel like we should have it now, I can do it too, but... I'd rather pay attention to other things, this little detour took way too much of my time yesterday 😅

Reply:
Sure, it can wait, thanks :slightly_smiling_face: